tox.ini

```diff
@@ -210,7 +215,7 @@ commands_pre =
   python -m pip install -e {toxinidir}/exporter/opentelemetry-exporter-otlp[test]
   python -m pip install -e {toxinidir}/exporter/opentelemetry-exporter-zipkin-json[test]
   python -m pip install -e {toxinidir}/exporter/opentelemetry-exporter-zipkin-proto-http[test]
-  python -m pip install -e {toxinidir}/exporter/opentelemetry-exporter-zipkin[test]
```

Comment:
This line probably shouldn't be removed.

Reply:
Fixed